pkgs/by-name/lo/local-ai/package.nix

```diff
@@ -27,11 +27,11 @@
   # https://github.com/NixOS/rfcs/pull/169
 
   # CPU extensions
-  enable_avx ? true,
-  enable_avx2 ? true,
+  enable_avx ? stdenv.hostPlatform.avxSupport,
```

Comment:
avxSupport is false on an default stdenv and llama is not usable with avx. We should use the isx86_64 flag.

Reply:
If it is not usable at all, there should be an assert and the platform list updated to x86_64-only

Reply:
I will add a comment with next version update.